test/test-cases/scale/saic/sai-api/test_sai_api_vnet_acl_in.py

```diff
@@ -0,0 +1,104 @@
+import json
+import time
+from pathlib import Path
+from pprint import pprint
+
+import pytest
+import saichallenger.dataplane.snappi.snappi_traffic_utils as stu
+from saichallenger.dataplane.ptf_testutils import (send_packet,
+                                                   simple_udp_packet,
+                                                   simple_vxlan_packet,
+                                                   verify_no_other_packets,
+                                                   verify_packet)
+
+import dash_helper.vnet2vnet_helper as dh
+
+current_file_dir = Path(__file__).parent
+
+# Constants
+SWITCH_ID = 5
+
+
+class TestSaiVnetAclIn:
+
+    def test_vnet_acl_in_create(self, dpu):
```

Comment:
More specifically, this is an IPv4 ACL group create. There are multiple aspects to making a useful "ACL" - creating a group, creating rule(s) and adding those rules to the group. So you should decide the scope of each test case (just create a group, or create a group and rules and add to the group); also consider the permutations of IPv4 and IPv6, etc.
Here is a good reference of PTF test cases: https://github.com/Azure/DASH/blob/746d482e2a022356bd82084c3f3e463343d004ac/test/test-cases/functional/ptf/saidashacl.py#L83